src/components/blog/cards/BlogRecommendationCard.tsx

```diff
@@ -17,7 +17,7 @@ export const BlogRecommendationCard = ({ blog }: { blog: Blog }) => {
       </Link>
 
       <div className='flex items-center gap-[6px]'>
-        <p className='font-roboto text-xs opacity-80'>
+        <p className='font-dm_sans text-xs opacity-80'>
```

Comment:
write the font in a global.css file, instead of specifying `font-dm_sans` every time, this will maintain the consistency for fonts everytime a new text is added

Reply:
Added roboto as base font in global.css.